v1/types/types.gunk

```diff
@@ -348,51 +348,54 @@ type Profile struct {
 	// Contact address
 	ContactAddress Address `pb:"13" json:"contact_address"`
 
+	// Contact number
+	ContactNumber string `pb:"14" json:"contact_number"`
+
 	// profile number
-	ProfileNumber string `pb:"14" json:"profile_number"`
+	ProfileNumber string `pb:"15" json:"profile_number"`
 
 	// Face image of the customer
-	FaceImageUrl string `pb:"15" json:"face_image_url"`
+	FaceImageUrl string `pb:"16" json:"face_image_url"`
 
 	// Date when the face image was added/updated
-	FaceImageDate string `pb:"16" json:"face_image_date"`
+	FaceImageDate string `pb:"17" json:"face_image_date"`
 
 	// RelationshipStatus. Ex: Single | Married
-	RelationshipStatus string `pb:"17" json:"relationship_status"`
+	RelationshipStatus string `pb:"18" json:"relationship_status"`
 
 	// Number of dependents
-	Dependents int `pb:"18" json:"dependents"`
+	Dependents int `pb:"19" json:"dependents"`
 
 	// Date of birth of dependents
-	DobOfDependents []time.Time `pb:"19" json:"dob_of_dependents"`
+	DobOfDependents []time.Time `pb:"20" json:"dob_of_dependents"`
 
 	// Credit rating
-	CreditRating CreditRating `pb:"20" json:"credit_rating"`
+	CreditRating CreditRating `pb:"21" json:"credit_rating"`
 
 	// Credit Limit
```

Comment:
please avoid changing the order/number/pb field position. it won't be backward compatible.

Reply:
as discussed, we agree that the field position changing at this stage is acceptable.